### PR TITLE
release/# prepare release 4.6.0

### PR DESCRIPTION
**Pull request by Nicoretti**
- Update lockfile
- Update dependabot configuration
- Update CI/CD status badge
- Update README.md
- Prepare release 4.6.0

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,6 +3,18 @@
...
+
+4.6.0 — 2023-07-17
+==================
+
+🚀 Feature
+-----------
+
+- Websocket based dialect have been stabilized and is officially supported now
+
+    📘 Note: Inserting multiple empty row's, facilitating default settings currently isn't supported.
```
Comment (tkilias) [inline CHANGELOG.rst:16]:
```suggestion
    📘 Note: Inserting multiple empty row's, facilitating default settings currently isn't supported by Exasol.
```

File: README.rst [hunk 2]
```diff
@@ -46,8 +46,13 @@ SQLAlchemy Dialect for EXASOL DB
 How to get started
 ------------------
 
-We assume you have a good understanding of (unix)ODBC. If not, make sure you
-read their documentation carefully - there are lot's of traps 🪤 to step into.
+Currently sqlalchemy-exasol supports multiple dialects. The core difference
+being if the dialect is :code:`odbc` or :code:`websocket` based.
+
+Generally we advise to use the websocket based Dialect, because odbc
```
Comment (tkilias) [inline README.rst]:
```suggestion
Generally, we advise to use the websocket based Dialect, because odbc
```

File: README.rst [hunk 3]
```diff
@@ -46,8 +46,13 @@ SQLAlchemy Dialect for EXASOL DB
 How to get started
 ------------------
 
-We assume you have a good understanding of (unix)ODBC. If not, make sure you
-read their documentation carefully - there are lot's of traps 🪤 to step into.
+Currently sqlalchemy-exasol supports multiple dialects. The core difference
```
Comment (tkilias) [inline README.rst]:
```suggestion
Currently, sqlalchemy-exasol supports multiple dialects. The core difference
```

File: CHANGELOG.rst (hunk 1 above)
Comment (tkilias) [reply CHANGELOG.rst:16]: or is this specific to the websocket implementation

**Review by tkilias — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (Nicoretti) [reply CHANGELOG.rst:16]: I would say it it specific to the dialect. Trubodbc does not work Pyodbc "supports it". For the websocket protocoll we need extra code to support it with other primitives.

**Review by tkilias — APPROVED** [review]

**Pull request merged**